backend/src/main/resources/liquibase/db-changelog-2.0.xml

```diff
@@ -137,7 +137,7 @@
         <tagDatabase tag="db_version_2.4"/>
     </changeSet>
 
-    <changeSet id="add-workspace-id-to-project" author="anandJ">
+    <changeSet id="add-workspace-id-to-project" author="cory">
```

Comment:
I did have to clean out my local DB and rebuild it from scratch.  That should not happen on master when we merge this though since only the final change should get applied (if I understand Liquibase properly...)

Reply:
fixed this in the base branch to avoid the has changing